proxy/pkg/proxy/proxy.go

```diff
@@ -86,6 +88,7 @@ func NewMultiHostReverseProxy(opts ...Option) *MultiHostReverseProxy {
 	for _, pol := range options.Config.Policies {
 		for _, route := range pol.Routes {
 			rp.logger.Debug().Str("fwd: ", route.Endpoint)
+			// TODO check either Backend or Service is set
```

Comment:
Could you implement the TODO now? It doesn't sound too big or difficult.

Reply:
Fair enough :+1:  Done.